tools/mob_suite/mob_typer.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="mob_typer" name="MOB-Typer" version="1.4.1">
+  <description>(Get the plasmid type and mobility given its sequence)</description>
+  <requirements>
+     <requirement type="package" version="1.4.1">mob_suite</requirement>
+     <!--requirement type="package" version="2.7.1">blast</requirement-->
+     <!--requirement type="package" version="2.0">mash</requirement-->
+     <!--requirement type="package" version="1.5.5">circlator</requirement-->
+  </requirements>   
+  <command detect_errors="exit_code">
+    ln -s ${input} ${input.name};
+    mob_typer  --num_threads 16 --infile "${input.name}" --evalue ${adv_param.evalue} --min_ident ${adv_param.min_ident} --min_cov ${adv_param.min_cov} --outdir ".";
+    ##/home/kbessonov/miniconda3/envs/general/bin/python /home/kbessonov/miniconda3/envs/general/bin/mob_typer  --num_threads 16 --infile "${input}" --outdir ".";
+    ##pwd > screen.out; ls >>screen.out; echo ${__tool_directory__} >> screen.out; 
+  </command>
+  <inputs>
+    <!--param name="num_lines" size="5" type="integer" value="1" label="Randomly select" help="lines"/-->
+    <param name="input" type="data" format="fasta" label="Input" help="FASTA file with contig(s)"/>
+    <section name="adv_param" title="Advanced parameters" expanded="False">
+      <param name="evalue" label="Minimum e-value threshold for BLAST" type="float"  min="0.000000000000000000001" max="1" value="0.00001"/>
+      <param name="min_ident" label="Minimum sequence identity for BLAST" type="float" min="1" max="100" value="80"/> 
+      <param name="min_cov" label="Minimum percentage coverage of assembly contig by the plasmid reference database to be considered" type="float" min="1" max="100" value="65"/> 
+    </section>  
+  </inputs>
+  <outputs>
+    <!--data name="outfile1" format="tabular" from_work_dir="mobtyper_${input.name}_report.txt" label="${tool.name} report from ${input.name}"/-->
+    <data name="outfile1" label="${tool.name}: Report from ${input.name}" format="tabular" hidden="false">
```

Comment:
If the user has an input.name that contains valid XML characters, such as `<, >` will this line be affected?

Reply:
corrected, thank you